server/grpc/api/mapping.go

```diff
@@ -65,7 +65,9 @@ func fromProtoParameters(params []*definition.Parameter) []*service.Parameter {
 
 func fromProtoConfiguration(configuration *definition.Configuration) *service.Dependency {
 	if configuration == nil {
-		return nil
+		return &service.Dependency{
```

Comment:
This will be updated in a future PR that i'm working one ;) so please don't block this PR because of this one.

Reply:
PR in question: https://github.com/mesg-foundation/engine/pull/1110